salt/modules/dracr.py

```diff
@@ -151,6 +151,107 @@ def __execute_ret(command, host=None,
     return cmd
 
 
+def get_property(host=None, admin_username=None, admin_password=None, property=None):
```

Comment:
When it comes to passing password directly to execution modules, IMHO that should always be pulled from pillars. This is going to show up in a process list while it's running and in shell history. :-/

Reply:
@johnybx Did you see the feedback above?

Reply:
@rallytime I was on vacation without computer.

@cachedout I followed other functions in that module. I could put there some condition if password is none check pillar data if admin_password exists there. Would this work for you ?

Reply:
If it's already in other functions then I'm fine with updating them all at once in another PR.